### ISSUE TITLE
pretrain model

### ISSUE THREAD

**Issue by ntzzc** [The-Learning-Machines/Audiomer-PyTorch#5]
hi, Surya Kant Sahu, good paper and work. I wanna try reproduce the exps. is it okay for you to share the pre-trained model?

**Comment by ojus1** [comment]
Hi, I don't have access to the machine which had the checkpoints right now. In the meantime, maybe @FredHaa or @yodakohl can upload their models and share here?

I'll try to upload our checkpoints asap.

**Comment by FredHaa** [comment]
I never managed to reach the accuracies you report in the paper. 
I got close with the S model, but as I reported in the other issue I sometimes get issues with NaN values. This still occurs even after your code refactor. I'm now using a fixed seed and hope to get the NaN values again. When that happens I can investigate the issue and hopefully fix it.

I also noticed a discrepancy between the batch size and learning rate you report in the preprint vs the code implementation. Did you use bs=128 and lr=2x10^3 or bs=64 and lr=5x10^3 when getting your results?

When that is said and done I'm also very interested in pretrained models as what I'm really trying to do is fine-tune on a custom dataset. If I didn't have to first try to train from scratch on SC35 it would be very helpful.

**Comment by pigpub** [comment]
@FredHaa have you changed any parameter in this repo? My experiment only get 93.4% on SC35 for the large model

**Comment by FredHaa** [comment]
I've changed a lot of things in the code base, but both with the original repo and my fork I reach those accuracies.

I faced similar problems as seen in this issue: https://github.com/The-Learning-Machines/Audiomer-PyTorch/issues/1

I've since then discovered that if I changed the root dir for the dataloader, then the dataset only contained 80k examples instead of the 110k examples which are in SC35. This caused my accuracy to drop to about 93% similar to you.

Try printing the length of the dataset with and without changing the root dir.
